### PR TITLE
Feature/cce 545

### PR DESCRIPTION
**Pull request by joseegm**

### CODE REVIEW

File: docu/release_notes/2016-10-28-Technev150.html.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+date: October 28, 2016
+service: 'hybris techne'
+official_version: v1.5.0
+headline: 'Techne v1.5.0 released, New Product Thumbnail component. Revisited Tree pattern.'
+---
+
+<h1>Techne v1.5.0</h1>
+<p>New Product Thumbnail component added. Can be used to show a list of products in a grid.</p>
```
Comment (saadmhybris) [inline docu/release_notes/2016-10-28-Technev150.html.md]: this should be pattern instead of component.

Comment (joseegm) [inline docu/release_notes/2016-10-28-Technev150.html.md]: I guess my 90's jargon comes up once in a while ;)

File: docu/release_notes/2016-10-28-Technev150.html.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+date: October 28, 2016
+service: 'hybris techne'
+official_version: v1.5.0
+headline: 'Techne v1.5.0 released, New Product Thumbnail component. Revisited Tree pattern.'
+---
+
+<h1>Techne v1.5.0</h1>
+<p>New Product Thumbnail pattern added. Can be used to show a list of products in a grid.</p>
+<p>New Grid Started pages added. Speeds the development of pages used to display products in a grid. It has also our new toolbar for easy filtering and navigation.</p>
```
Comment (saadmhybris) [inline docu/release_notes/2016-10-28-Technev150.html.md]: "New Product Grid starter page added to show Product Thumbnails in a Gird"

I think we should just leave it at that. the mention of speed and toolbar may cause confusion and take focus away from the new shiny thing as we already released the toolbar a while ago.

File: docu/release_notes/2016-10-28-Technev150.html.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+<p>Revisited Tree pattern, along with respective Started page. Now with more standard expand/collapse icons and behavior. Description of how the interactivity should work.</b>
+
```
Comment (saadmhybris) [inline docu/release_notes/2016-10-28-Technev150.html.md]: Let's break this down into two bullet items one for the pattern and another for the starer page 

- Updated Tree pattern. Now with more standard expand/collapse icons and behavior and description of how the interactivity should work.
- New Tree Starter Page added to show the interactive behaviour and rules in action

File: docu/release_notes/2016-10-28-Technev150.html.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+<h3>Minor updates</h3>
+<ul>
+    <li>Improved Icons page now displaying the icons in a more easy to glance grid. Added suggestions on how to use colors on icons</li>
```
Comment (saadmhybris) [inline docu/release_notes/2016-10-28-Technev150.html.md]: Improved Icons page with grid layout for easy scanability. Added suggested colors for each icon based on it's recommended usage

File: docu/release_notes/2016-10-28-Technev150.html.md (hunk 3 above)
Comment (joseegm) [inline docu/release_notes/2016-10-28-Technev150.html.md]: Changed

File: docu/release_notes/2016-10-28-Technev150.html.md (hunk 4 above)
Comment (joseegm) [inline docu/release_notes/2016-10-28-Technev150.html.md]: Changed

**Pull request merged**